pages/declaration.js

```diff
@@ -74,3 +75,8 @@ const Declaration = props => (
 );
 
 export default SessionWrapper(Declaration);
+
+Declaration.propTypes = {
+  cumalativeAnswers: PropTypes.objectOf(PropTypes.string),
```

Comment:
typo of `cumulativeAnswers`

Reply:
Looks like this is across the board, just need to do a global find and replace for `cumalativeAnswers` > `cumulativeAnswers`